test/ValidateOAuthVersionMessageHandler.cs

```diff
@@ -10,63 +10,37 @@ public class ValidateOAuthVersionMessageHandler
         [Fact]
         public async Task ValidateDefaultOAuthVersionHandler()
         {
-            using (var handler = new OAuthMessageHandler("apiKey", "secret", "authToken", "authTokenSecret"))
-            {
-                var oauthVersion = await GetOAuthVersionFromHandlerAsync(handler);
+            TestHelper th = new TestHelper("apiKey", "secret", "token", "tokenSecret");
 
-                Assert.Equal("1.0", oauthVersion);
-            }
+            Assert.Equal("1.0", await th.ComputeOAuthVersionAsync(GetDummyRequest(), "nonce", "timestamp", OAuthVersion.OneZero));
```

Comment:
This test no longer tests that the default value for `oauth_version` is 1.0, since the `TestHelper` explicitly sets the version in the call to the constructor of `OAuthMessageHandler`. Also, the code of this test is now identical to the code of `ValidateOAuthVersionOneZeroHandler`.

Reply:
Good point. I don't see a good way to validate the default. The way I am thinking now is to go back to reflection :). Basically, create an `OAuthMessageHandler` without specifying the version, use reflection to get to the `IOAuthSignatureData` provider and then checking that it is "1.0". 

Let me know if you have a better idea, otherwise I'll go with this one.

Reply:
That would be one idea, another would be to essentially do the same thing that `TestHelper.ComputeOAuthVersionAsync` does, but calling the `OAuthMessageHandler(string apiKey, string secret, string authToken, string authTokenSecret)` constructor instead of the one taking an `IOAuthSignatureData`. I'm not sure on the best way to do something like that (sending an `OAuthMessageHandler` to the `TestHelper`, calling different constructors based on arguments, add a new method to `TestHelper`, et.c.) but it could be argued that we should also test the constructor taking in `OAuthVersion` so the solution should maybe support that as well.

Reply:
I went with the test helper approach. It was in-line with the current approach and easy to understand and code.